ethers-contract/ethers-contract-abigen/src/contract/events.rs

```diff
@@ -230,19 +230,7 @@ pub(crate) fn event_struct_alias(event_name: &str) -> Ident {
 mod tests {
     use super::*;
     use crate::Abigen;
-    use ethers_core::abi::{EventParam, Hash, ParamType};
-    use proc_macro2::Literal;
-
-    /// Expands a 256-bit `Hash` into a literal representation that can be used with
```

Comment:
is this going somewhere? is dead code?

Reply:
I couldn't understand it; it's a local function in the test module, so it's not used anywhere else and had a test.. it doesn't really do anything